ui-tests/src/test/java/org/openmrs/reference/DeleteAllergyTest.java

```diff
@@ -31,12 +31,12 @@
     @Before
     public void setUp() throws Exception {
         patient = createTestPatient();
-        createTestVisit();
+        new TestData.TestVisit(patient.uuid, TestData.getAVisitType(), getLocationUuid(homePage)).create();
```

Comment:
Thanks @kdaud  , why are we adding this createTestVisit methd in the before part

Reply:
> why are we adding this createTestVisit methd in the before part , what happens if we dont touch that part

As far as I know **we definitely need this method**. Its not a good to expose out test to any of the data-set that could be present within the server. So this method ensures data-set creation and the test work flow for which the test will consume rather than using any external data. cc: @sherrif10